packages/itinerary-body/src/styled.tsx

```diff
@@ -294,6 +290,14 @@ export const LegDescription = styled.span`
   min-height: 31px;
 `;
 
+// additional description added to ClickableLeg for screenreaders
+export const AccessibilityLinkDetails = styled.span`
```

Comment:
I love this! But I'd maybe make the name more clear -- it should be explicit that this is hidden text for screen readers

Reply:
Good call!